synapse/server/nodes/stream_out.py

```diff
@@ -34,46 +29,32 @@ def config(self):
         return c
 
     def configure(self, config: StreamOutConfig) -> Status:
-        if not self.__iface_ip:
-            return Status(StatusCode.kUndefinedError, "No interface IP specified")
-
-        if not config.multicast_group:
-            return Status(StatusCode.kUndefinedError, "No multicast group specified")
-
         self.__config = config
 
+        if not config.udp_unicast:
+            self.logger.error(
+                "Cannot conifgure StreamOut, only udp unicast is supported"
+            )
+            raise Exception("Only udp unicast is supported for streamout")
+
         self.__socket = socket.socket(
             socket.AF_INET, socket.SOCK_DGRAM, socket.IPPROTO_UDP
         )
-
         self.__socket.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
 
-        port = PORT + self.__i
+        dest_address = self.__config.udp_unicast.destination_address
+        dest_port = self.__config.udp_unicast.destination_port
 
-        self.__socket.bind((self.__iface_ip, port))
-
-        mreq = struct.pack(
-            "=4sl", socket.inet_aton(config.multicast_group), socket.INADDR_ANY
-        )
-        self.__socket.setsockopt(socket.IPPROTO_IP, socket.IP_ADD_MEMBERSHIP, mreq)
-        self.__socket.setsockopt(
-            socket.IPPROTO_IP, socket.IP_MULTICAST_TTL, MULTICAST_TTL
-        )
-
-        self.socket = [config.multicast_group, port]
-        self.logger.info(
-            f"created multicast socket on {self.socket}, group {config.multicast_group}"
-        )
+        self.__socket.bind((dest_address, dest_port))
```

Comment:
I believe we don't need to bind to send, and we don't want to bind to the dest port if we want to receive on the same machine.